javascript/javascript1/week3/series-duration.js

```diff
@@ -0,0 +1,59 @@
+const seriesDurations = [
+    {
+        title: "Game of thrones",
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+    {
+        title: "Sopranos",
+        days: 3,
+        hours: 14,
+        minutes: 0,
+    },
+    {
+        title: "The Wire",
+        days: 2,
+        hours: 12,
+        minutes: 0,
+    },
+    {
+        title: "Friends",
+        days: 4,
+        hours: 2,
+        minutes: 20,
+    },
+    {
+        title: "How i met your mother",
+        days: 7,
+        hours: 6,
+        minutes: 36,
+    },
+    {
+        title: "House of cards",
+        days: 7,
+        hours: 10,
+        minutes: 16,
+    },
+    {
+        title: "Money heist",
+        days: 8,
+        hours: 22,
+        minutes: 26,
+    },
+];
+const totalMinutes = 365.25 * 80 * 24 * 60;
+
+function CalAvg(seriesDurations) {
+    let totalTimeSpend = 0;
+    for (let i = 0; i < seriesDurations.length; i++) {
+        let minutes = (seriesDurations[i].days * 24 * 60) + (seriesDurations[i].hours * 60) + seriesDurations[i].minutes;
+        let timeSpend = minutes * 100 / totalMinutes;
```

Comment:
Since `minutes` and `timeSpend` are assigned within the `for` loop it is safe to use `const` here as well. They are "constant" for a single iteration of the loop and forgotten at the end (`}`), just to be created anew when the `for` loop start a new iteration.

Does it make sense?

Reply:
done it.

Reply:
yes, it make sense. done it